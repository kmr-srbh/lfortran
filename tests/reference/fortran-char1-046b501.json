{
    "basename": "fortran-char1-046b501",
    "cmd": "lfortran --show-fortran --no-color {infile} -o {outfile}",
    "infile": "tests/char1.f90",
    "infile_hash": "78e727aecb70503bac5e780ad046d418a533f576cf7b276702bb6be1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "fortran-char1-046b501.stderr",
    "stderr_hash": "7b2492beb9564b9bf748dd9c9dc5dca07daff9d2d1845aff1fafb877",
    "returncode": 1
}