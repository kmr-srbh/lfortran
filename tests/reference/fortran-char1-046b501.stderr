semantic error: The RHS character len must be known at compile time
 --> tests/char1.f90:2:1
  |
2 | character(len=1, kind=1), parameter :: c_null_char = char(0)
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
